test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -373,7 +373,7 @@ public void reloadCore(int node, String keyspace, String table, boolean reindex)
   public void start() {
     if (started.compareAndSet(false, true)) {
       try {
-        execute("start", jvmArgs, "--wait-for-binary-proto", "--wait-other-notice");
+        execute("start", jvmArgs, "--wait-for-binary-proto");
```

Comment:
Why did you remove it ?
It will change CCM behavior for all tests and can cause tests instability.

Reply:
It clashes with `join_ring` set to false. It will timeout. IIRC we added this on a whim not that long ago, so I just removed it here. Other option would be making this configurable, so that it can be disabled just for zero token tests.

Reply:
Let's add another public method, startNoWait and use it for `join_ring=false` nodes

Reply:
Added `startWithArgs` that uses custom arguments. Old `start` should be functionally the same as before.